### PR TITLE
EPMEDUTOUR-10381: Simplified Chenese support

### PR DESCRIPTION
**Pull request by error404as**
### What does this PR do:

Added Simplified Chinese and sections language fallbacks if no translations yet for it.

### CODE REVIEW

File: src/_data/sectionAbout.json [hunk 1]
```diff
@@ -48,5 +48,30 @@
                 "body": "Plan your trip and create lists of your favorite places"
             }
         ]
+    },
+    "ch": {
```
Comment (alexeykomov) [inline src/_data/sectionAbout.json]: Do we use simplified Chinese here?
If so, let's put `zh-Hans` as lang code.
If we're not sure, let's use `zh`.

P.S. We use simplified chinese for mobile app.
Screenshot 2022-08-19 at 15 53 08

**Review by alexeykomov — COMMENTED** [review]
I think it should be `zh` or `zh-Hans` here.

Screenshot 2022-08-19 at 15 56 16

I used [this reference](https://www.w3.org/International/articles/language-tags/) for language tags.
> Macrolanguages The primary language subtags used with an extlang subtag are known as macrolanguages, and encompass a number of languages with more specific primary language subtags. The macrolanguage subtag can be used on its own, but unless there is some convention about its meaning in the context where it is used, it is not necessarily precise enough.

> For example, zh means Chinese, but it covers many Chinese dialects, often mutually incomprehensible. When zh is used on its own, it is usually used to mean the predominant language in the encompassed range, although this is not explicitly specified in BCP 47. For example, conventionally zh is considered to represent the predominant, Mandarin form of Chinese. Where absolute clarity is needed you can use cmn instead as long as that doesn't break interoperability, however, if you are using zh to represent a language which is not Mandarin, such as Hakka Chinese, you are better off using the explicit code (in that case, hak).

> On the other hand, zh-Hans uses zh in its generic sense. This is a useful way to describe writing in Simplified Chinese, since Chinese tends to be written in the same way, regardless of the dialect of the reader.

**Review by alexeykomov — APPROVED** [review]

File: src/_components/altlang.ejs [hunk 2]
```diff
@@ -2,8 +2,7 @@
 var langs = Object.keys(lang).filter(el => el !== locale);
 var pattern = '^(\/' + locale + ')';
 var reg = new RegExp(pattern, '');
-var altpath = '/' + altlang;
 %>
 <% langs.forEach((lang) => { %>
-<link href="<%= page.url.replace(reg, lang) %>" rel="alternate" hreflang="<%= lang %>">
+<link href="/<%= page.url.replace(reg, lang) %>" rel="alternate" hreflang="<%= lang === 'ch' ? 'zh' : lang %>">
```
Comment (alexeykomov) [inline src/_components/altlang.ejs]: Maybe let's use `zh` here as well?

File: src/_data/site.json [hunk 3]
```diff
@@ -1,12 +1,16 @@
...
+  "defaultLocale": "ru",
   "en": {
-    "description": "Site meta description"
+    "description": "TBD_en_description"
   },
   "ru": {
-    "description": "Описание страницы в мета"
+    "description": "TBD_ru_description"
+  },
+  "ch": {
```
Comment (alexeykomov) [inline src/_data/site.json]: Maybe let's use `zh` here as well?

**Review by alexeykomov — APPROVED** [review]

**Review by ashumski-adnoc — APPROVED** [review]

**Pull request merged**